src/core/maps/_core-fonts-map.scss

```
@@ -0,0 +1,35 @@
+// COR MAP: FONTS
+// This partial defines font-related maps.
+// - Maps include Font scope tokens.
+// - Maps are collected into the $core-settings-map and are interpolated into custom properties.
+
+$core-fonts-map: (
+        font-family:$font-family,
```

nit-picking: it really bothered me that this file has no space after the colon 🙈 
```suggestion
        font-family: $font-family,
```

---

you are so right.
i will fix it on next PR. :)

---

why not pushing an update as part of this PR? :)